src/functions/Publish-PuppetModule.ps1

```diff
@@ -0,0 +1,63 @@
+function Publish-PuppetModule {
+    <#
+      .SYNOPSIS
+        Build and Publish Module
+      .DESCRIPTION
+        Generate package for the Puppet module and publish to the forge.
+      .PARAMETER PuppetModuleFolderPath
+        The path, relative or literal, to the Puppet module's root folder.
+      .PARAMETER ForgeUploadUrl
+        The Forge Upload Url Path and default is https://forgeapi.puppetlabs.com/v3/releases.
+      .PARAMETER ForgeToken
+        The Forge Token.
+      .PARAMETER Build
+        Flag to build the package.
+      .PARAMETER Publish
+        Flag to publish the package.
+      .EXAMPLE
+        Publish-PuppetModule -PuppetModuleFolderPath C:\output\testmodule -ForgeUploadUrl https://forgeapi.puppetlabs.com/v3/releases -ForgeToken testmoduletoken -Build true -Publish true
+        This command will createo or use existing pkg and Publishes the <tarball> to the Forge , for the `testmodule` depends on the options passed for pdk release command.
+    #>
+    #>
+    [CmdletBinding()]
+    param (
+      [Parameter(Mandatory=$True)]
+      $PuppetModuleFolderPath,
+      [string]$ForgeToken,
+      [string]$ForgeUploadUrl,
+      [string]$Build,
+      [string]$Publish
```

Comment:
These two should be boolean, not string flags. this makes the code below a lot easier.

See also https://stackoverflow.com/a/10581229/4918